src/main/java/com/urbanairship/connect/client/Stream.java

```diff
@@ -44,6 +44,9 @@
 
     public Stream(StreamQueryDescriptor descriptor, Optional<StartPosition> startingPosition) {
         this(descriptor, startingPosition, Optional.<StreamConnectionSupplier>absent());
+        log.debug("Starting Stream. Filters: " + descriptor.getFilters()
```

Comment:
might be a nit, but the stream isn't actually started by this call-- I'd avoid 'Starting' until you actually start streaming events.

Reply:
Updated!